packages/account/package.json

```diff
@@ -39,24 +39,24 @@
         "js-cookie": "^2.2.1",
         "mobx": "^6.6.1",
         "mobx-react": "^7.5.1",
-        "onfido-sdk-ui": "8.1.1",
+        "onfido-sdk-ui": "^11.0.0",
```

Comment:
Since you changed the version of these packages could u pls regenerate package-lock too :pray:

Reply:
by running `npm run bootstrap:dev`

Reply:
I have regenerated. Thank you